sdk/digitaltwins/digital-twins-core/CHANGELOG.md

```diff
@@ -2,6 +2,7 @@
 
 ## 1.0.2 (Unreleased)
```

Comment:
```suggestion
## 1.0.2 (2021-01-14)
```

Reply:
Thanks, I wasn't sure if the bot updates this as well